src/applications/disability-benefits/2346/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,101 @@
+import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import { focusElement } from 'platform/utilities/ui';
+import React from 'react';
+
+class IntroductionPage extends React.Component {
+  componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Order hearing aid batteries and prosthetic socks from VA" />
+        <p>Equal to VA Form 2346 (Request for Batteries and Accessories).</p>
+        <SaveInProgressIntro
+          hideUnauthedStartLink
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
+          messages={this.props.route.formConfig.savedFormMessages}
+          pageList={this.props.route.pageList}
+          startText="Start the Application"
+        >
+          Please complete the 2346 form to apply for ordering hearing aid
+          batteries and accessories.
+        </SaveInProgressIntro>
+        <h2 itemProp="name" id="am-i-eligible-to-order-prosthe">
+          Follow the steps below to order hearing aid batteries aid accessories.
+        </h2>
+        <div className="process schemaform-process">
+          <ol>
+            <li className="process-step list-one">
+              <h3>Prepare</h3>
+              <p>To place an order, you'll need your:</p>
+              <ul>
+                <li>Shipping address</li>
+                <li>Email address</li>
+                <li>Hearing aid information</li>
+              </ul>
+              <h4>What if I need help with my order?</h4>
+              <p>
+                If you need help ordering hearing aid and betteries and
+                accessories, you cna contact the Devner Logistics Center (DLC)
+                at 303-273-6200
```

Comment:
Please wrap the telephone number in a `<a href="tel:1-303-273-6200">....</a>`

Reply:
Done